impl/src/bitfield/expand.rs

```diff
@@ -377,6 +377,7 @@ impl BitfieldStruct {
                 ReprKind::U128 => quote! { IsU128Compatible },
             };
             quote_spanned!(span=>
+                #[allow(unused_braces)]
```

Comment:
please write a short comment describing why we are silencing this warning here

Reply:
Done